CRM/Certificate/Form/CertificateConfigure.php

```diff
@@ -279,4 +321,28 @@ public function validateParticipantTypeField($values, &$errors) {
     }
   }
 
+  /**
+   * Validates download format field.
+   *
+   * @param array $values
+   * @param array $errors
+   */
+  public function validateFormatType($values, &$errors) {
+    if ($values['download_format'] == "2" && empty($values['image_format_id'])) {
```

Comment:
If you as I mentioned above store `const PDF = '1';` and `const IMAGE = '2';` inside CRM_Certificate_BAO_CompuCertificate class as an enumeration, you can use it here instead of having "2" hardcoded, which would make things more readable.